packages/hash/api/src/seed-data/dev-users.ts

```diff
@@ -43,13 +46,43 @@ export const ensureDevUsersAreSeeded = async ({
 }): Promise<UserModel[]> => {
   const createdUsers = [];
 
-  for (const { email, shortname, preferredName, isInstanceAdmin } of devUsers) {
+  // Only use `devUsers` if we are in a dev environment
+  let usersToSeed = !isDevEnv ? [] : devUsers;
+
+  // Or if we're explicitly setting users to seed.
+  if (process.env.HASH_SEED_USERS) {
+    try {
+      usersToSeed = JSON.parse(
+        process.env.HASH_SEED_USERS,
+      ) as DevelopmentUser[];
```

Comment:
We should probably rename `DevelopmentUser` to `SeededUser` or something as it's no longer just used in development now